website/forms/people_filter.py

```diff
@@ -3,45 +3,45 @@
 from wtforms.validators import InputRequired
 
 class OrganizationCheckbox(FlaskForm):
-    individual = BooleanField(label="Individual", name="individual", render_kw={"class": ""})
-    group = BooleanField(label="Group", name="group", render_kw={"class": ""})
-    organization = BooleanField(label="Organization", name="organization", render_kw={"class": ""})
+    individual = BooleanField(label="Individual", name="individual", render_kw={"class": "form-check-input"})
+    group = BooleanField(label="Group", name="group", render_kw={"class": "form-check-input"})
+    organization = BooleanField(label="Organization", name="organization", render_kw={"class": "form-check-input"})
 
 class FiltersCheckbox(FlaskForm):
-    choreographer = BooleanField(label="Choreographer", name="choreographer", render_kw={"class": ""})
-    designer = BooleanField(label="Designer", name="designer", render_kw={"class": ""})
-    writer = BooleanField(label="Writer", name="writer", render_kw={"class": ""})
-    producer = BooleanField(label="Producer", name="producer", render_kw={"class": ""})
-    stageManager = BooleanField(label="Stage Manager", name="stageManager", render_kw={"class": ""})
-    other = BooleanField(label="Other", name="other", render_kw={"class": ""})
+    choreographer = BooleanField(label="Choreographer", name="choreographer", render_kw={"class": "form-check-input"})
+    designer = BooleanField(label="Designer", name="designer", render_kw={"class": "form-check-input"})
+    writer = BooleanField(label="Writer", name="writer", render_kw={"class": "form-check-input"})
+    producer = BooleanField(label="Producer", name="producer", render_kw={"class": "form-check-input"})
+    stageManager = BooleanField(label="Stage Manager", name="stageManager", render_kw={"class": "form-check-input"})
+    other = BooleanField(label="Other", name="other", render_kw={"class": "form-check-input"})
 
 class PeopleFilter(FlaskForm):
-    searchName = StringField(
```

Comment:
@prabjot09 Please note I changed the variable names for some of these to make them the same as the other filter form, to make making it modular more simple to do. You might need to update code you've worked on that uses these variables.